### PR TITLE
test: Fix tests not running in VS when .NET 7 is installed

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
Thanks!

**Review by MartinZikmund — APPROVED** [review]

**Comment by jeromelaban** [comment]
This looks like the change made here has an impact on zip generation, somehow? I do not see how that's possible though. It really looks like an azure devops issue.

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -60,7 +60,7 @@
 	<PackageReference Update="Uno.Wasm.WebSockets" Version="1.1.0" />
 	<PackageReference Update="Microsoft.TypeScript.MSBuild" Version="4.3.5" />
 	<PackageReference Update="NUnit" Version="3.13.3" />
-	<PackageReference Update="NUnit3TestAdapter" Version="4.2.1" />
+	<PackageReference Update="NUnit3TestAdapter" Version="4.3.0-alpha-net7.4" />
```
Comment (carldebilly) [inline src/Directory.Build.targets:63]: Wow alpha ;-)

**Review by carldebilly — APPROVED** [review]

**Comment by Youssef1313** [comment]
@jeromelaban The test that was failing here is `When_Transforms_Paused_With_FillBehaviorHold_Then_Hold` which also failed previously and restarting it didn't resolve the issue, see https://github.com/unoplatform/uno/pull/9981#issuecomment-1259431249.

**Comment by MartinZikmund** [comment]
@Youssef1313 I suggest temporarily disabling the test and creating an issue for it (or put it in flaky test roundup if flaky)

**Comment by Youssef1313** [comment]
I think we could just merge for now. If the test shows as problematic again, we can disable.

I'll wait for @jeromelaban

**Pull request merged**